doc/source/cluster/kuberay/quickstart.md

```diff
@@ -0,0 +1,111 @@
+(kuberay-quickstart)=
+
+# Quickstart
+
+:::{warning}
+This page is under construction!
+:::
+
+Here we describe how you can deploy an autoscaling Ray cluster on KubeRay. The following instructions are for
+Minikube but the deployment works the same way on a real Kubernetes cluster. You need to have at
+least 4 CPUs to run this example. First we make sure Minikube is initialized with
+
+```shell
+minikube start
+```
+
+Now you can deploy the KubeRay operator using
+
+```shell
+./ray/python/ray/autoscaler/kuberay/init-config.sh
```

Comment:
Will get rid of this shell-script in a follow-up. The main purpose of this script was to check out a fixed KubeRay commit from the master branch.
The instructions will instead point to KubeRay 0.3.0 config resources.